### ISSUE TITLE
how to make .FEM meshes without houdini.

### ISSUE THREAD

**Issue by Khyretos** [GPUOpen-Effects/FEMFX#12]
Hey there,

is it possible for you guys to make a tutorial on how to make .FEM models on Blender
i am currently able to open the project but i cannot seem to be able to make any meshes

**Comment by Tetrahedrite** [comment]
@kireita Did you get any good idea?

**Comment by Khyretos** [comment]
no, this was 2 years ago, unfortunatly this project is dead. you can probably rip the files and look for the extensions but i cant help anymore. im sorry.
